frontend/src/app/modules/workspace/sidebar/sidebar.component.html

```diff
@@ -1,38 +1,59 @@
 <mat-nav-list>
-    <a class="logo" routerLink="">
-        <img src="../../../../assets/icons/logo-white.svg" alt="logo" />
-    </a>
+  <a class="logo" routerLink="">
+    <img src="../../../../assets/icons/logo-white.svg" alt="logo" />
+  </a>
 
-    <div class="workspace-container">
-        <div class="workspace-user">
-            <span class="user-subheader">
-                {{this.user.workspaceName}}
-            </span>
-            <span class="member-subheader">
-                1 member
-            </span>
-        </div>
-        <button mat-button class="workspace-button" routerLink="">
-            Invite teammates
-        </button>
+  <div class="workspace-container">
+    <div class="workspace-user">
+      <mat-form-field class="teams-box" appearance="fill">
+        <mat-label>Team:</mat-label>
```

Comment:
Can you make it matching the background? Because colors are still a little different
![image](https://user-images.githubusercontent.com/20342130/160063404-90abe29c-598e-4eaa-a9a0-ff26369dc3fe.png)